### ISSUE TITLE
Unexpected error comes out when everything is ok (related to jsx-indet)

### ISSUE THREAD

**Issue by SidKwok** [yannickcr/eslint-plugin-react#1118]
This is ok and no error comes out.
```jsx
    return (
      <p>
        I'm {this.state.name}
      </p>
    )
```

when I add any character after this like:
```jsx
    return (
      <p>
        I'm {this.state.name} any string here
      </p>
    )
```
and the error comes out:
```
Expected indentation of 8 space characters but found 1. (react/jsx-indent)
```

my eslint config : 
```javascript
module.exports = {
  root: true,
  parser: 'babel-eslint',
  parserOptions: {
    sourceType: 'module',
    ecmaFeatures: {
      jsx: true
    }
  },
  env: {
    es6: true,
    commonjs: true,
    browser: true,
  },
  extends: [
      'standard',
      'standard-react'
  ],
  plugins: [
    'react',
    'babel',
    'promise'
  ],
  'rules': {
    'arrow-parens': 0,
    'generator-star-spacing': 0,
    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0
  }
}

```

Is it a bug or something wrong with my config?

**Comment by ivantcov** [comment]
Same for me, failing e.g. on the following line:
`<Time fromNumber={this.state.value}/>&nbsp;<Translate for="shortHours"/>`

Not sure from which version it start happening.

**Comment by SidKwok** [comment]
The version I used: ^6.10.2

**Comment by benperiton** [comment]
Yep, seems to be counting the first group of space after a react element. (6.10.2)

```
<Icon name="tasks" fixedWidth /> Create Order
```
Gives 1

```
<Icon name="tasks" fixedWidth />   Create Order
```
Gives 3

```
<Icon name="tasks" fixedWidth />Create Order
```
Gives 0

**Comment by megapctr** [comment]
Temporary fix: use a fixed version until the bug is fixed. `6.10.0` works fine

**Comment by SidKwok** [comment]
@megapctr it is not working for me

**Comment by aliaksandr-master** [comment]
I have 6.10.2 version and it doesn't work
```js
          <div className="PageCampaignEditSettingsAdExample__headRating">
            <Rating rating={rating} />
            ( {reviews} )
          </div>
```
> Expected indentation of 12 space characters but found 1  react/jsx-indent

**Comment by ljharb** [comment]
This is a duplicate of #1117 - thanks for the extra test cases!
